cg/apps/housekeeper/hk.py

```diff
@@ -78,6 +78,34 @@ def delete_file(self, file_id: int) -> Optional[models.File]:
 
         return file_obj
 
+    def delete_files(
+        self, dry_run: bool = False, bundle: str = None, tags: List[str] = None, version: int = None
+    ) -> None:
+        """Delete files from database and disk (if included)"""
+        files: Iterable[File] = self.files(bundle=bundle, tags=tags, version=version)
+        for file in files:
+            file_obj_path = Path(file.full_path)
+            if (
+                not dry_run
```

Comment:
It would be great to extend this to delete any files, not just included, and move "only_included=True" into argument. That way nobody needs to make new method to delete also not included files!

Reply:
also I now see that we already have delete_file method. You could reuse it in a loop over list of files that comes back from files() method while checking included status

Reply:
agreed. I might change the function all together to work on the models File.full_path instead, but until then I will implement this change!